### ISSUE TITLE
about experiment results

### ISSUE THREAD

**Issue by fkxssaa** [JaywongWang/DenseVideoCaptioning#5]
Thank you for your contribution! I've learned a lot from your code. I pre-trained proposal model for 5 epochs. And trained the whole model for 40 epochs on a TITAN Xp. But the METEOR score on val dataset is about 8.5.  Could you please analyze possible reasons why I did not reach the result you reported.

**Comment by JaywongWang** [comment]
One reason could be using different random seed and thus different initialization. By the way, have you try the joint ranking technique? You may need to first generate .json file using max_proposal_num=1000 then select the top 100 results for each video. You may want to check the updated README.md.

**Comment by JaywongWang** [comment]
And also make sure you properly set eval_metric_num=5000 (any number that is greater than the size of validation set) to test on all validation samples.
